.github/workflows/try.yml

```diff
@@ -20,15 +28,34 @@ jobs:
         with:
           clean: False
 
-      - name: Clone qibo repository
-        uses: actions/checkout@v2
+      #- name: Clone qibo repository
+      #  uses: actions/checkout@v2
+      #  with:
+      #    repository: qibogang/qibo
+
+      - name: Deploy documentation
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[docs]
+          sudo apt install pandoc
```

Comment:
Do we really need `pandoc`?

Reply:
I tried without it, but the workflow gave errors.

Reply:
Try to understand why, link the errors here, and then you can also mark the conversation as solved

Reply:
The error is the following one (see [job](https://github.com/qibogang/qibo/actions/runs/3479064061/jobs/5817231526`)) : 

> Notebook error:
> PandocMissing in code-examples/tutorials/qPDF/qPDF.ipynb:
> Pandoc wasn't found.
> Please check that pandoc is installed:
> https://pandoc.org/installing.html
> make: *** [Makefile:20: html] Error 2
> Error: Process completed with exit code 2.

It appears that `pandoc` must be installed for `nsfinx` to work.